runtime/int.go

```diff
@@ -98,6 +98,22 @@ func intEq(f *Frame, v, w *Object) (*Object, *BaseException) {
 	return intCompare(compareOpEq, toIntUnsafe(v), w), nil
 }
 
+func intFloorDiv(f *Frame, v, w *Object) (*Object, *BaseException) {
+	if !v.isInstance(IntType) {
+		return nil, f.RaiseType(TypeErrorType, fmt.Sprintf("descriptor '__floordiv__' requires a 'int' object but received a '%s'", v.typ.Name()))
+	}
+
+	if w.isInstance(IntType) {
+		return intDivModOp(f, "__floordiv__", v, w, func(v, w int) (int, divModResult) {
+			return intCheckedDiv(v, w)
+		}, func(z, x, y *big.Int) {
+			longDiv(z, x, y)
+		})
+	}
+	fv, _ := intFloat(f, v)
```

Comment:
You should be able to return NotImplemented in this case and it will delegate to w's operator.